### PR TITLE
Hook up Twinkles to Travis CI using conda install of the stack + lsst_sims

### PR DESCRIPTION
**Pull request by jchiang87**
This addresses #169

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+language: C
+
+install:
+  - CHANNEL=http://eupsforge.net/conda/dev ./setup/install.sh lsst-sims
```
Comment (mjuric) [inline .travis.yml]: I'd propose to change this to:
```
CHANNEL=http://eupsforge.net/conda/dev ./setup/install.sh lsst-sims nose pandas
```
That way you don't need `requirements.txt`, and once pandas & nose are installed they'll be efficiently cached together with the rest of Python, etc.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - source eups-setups.sh
+  - setup lsst_sims
+  - pip install -r setup/requirements.txt
```
Comment (mjuric) [inline .travis.yml]: If you implement the suggestion above, you can remove this line (and `requirements.txt`).

Comment (jchiang87) [inline .travis.yml]: Awesome! Thanks!

File: .travis.yml (hunk 1 above)
Comment (jchiang87) [inline .travis.yml]: So I changed the line as suggested (86c0f15), and the build failed with this:
```
+conda config --add channels http://eupsforge.net/conda/dev
+conda install -q --yes 'lsst-sims nose pandas'
Error: Invalid package specification: lsst-sims nose pandas
The command "CHANNEL=http://eupsforge.net/conda/dev ./setup/install.sh lsst-sims nose pandas" failed and exited with 1 during .
Your build has been stopped.
```

Comment (jchiang87) [inline .travis.yml]: Removing the double quotes around `$PACKAGE` at [line 59](https://github.com/jchiang87/Twinkles/blob/u/jchiang/connecting_to_travis-ci/setup/install.sh#L59) of `install.sh` fixed the problem.

**Comment by mjuric** [comment]
@jchiang87 I fixed the mistake with the quoting on the current master (https://github.com/mjuric/lsst-travis-demo/blob/master/ci/install.sh), slightly differently than you did (I wanted to ensure any quoting or spaces in arguments survive and get passed on to `conda install`). It should be a drop-in replacement of the current `install.sh` file.

**Comment by jchiang87** [comment]
Thanks @mjuric, I'll integrate your changes later today.  I'm wondering if it would be possible to "centralize" this script somehow so that updates are propagated to everyone's packages automatically, i.e., grab `install.sh` from your repo instead of using a local copy.

**Comment by mjuric** [comment]
Yes, I've been thinking about how to do that, but I'm not sure how I'd do that (need to think more). Copying should be OK for now.

**Comment by jchiang87** [comment]
Phil says ok, let's merge!

**Pull request merged**